--- /Users/filippo/Downloads/oath-toolkit.rb	2023-06-06 13:08:53
+++ oath-toolkit.rb	2023-06-06 13:23:34
@@ -24,8 +24,12 @@
     sha256               x86_64_linux:   "3ea5398bb38297c062a54cce0ee803487211cf9f16c7697981d67a3edebd94e6"
   end
 
+  pour_bottle? do
+    reason "This is a patched formula to remove dependencies."
+    satisfy { false }
+  end
+
   depends_on "pkg-config" => :build
-  depends_on "libxmlsec1"
 
   def install
     system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
